config/default/core.entity_view_display.node.page.default.yml

```diff
@@ -59,8 +59,8 @@ third_party_settings:
         layout_settings:
           label: Header
           layout_builder_styles_style:
-            section_margin_edge_to_edge: section_margin_edge_to_edge
-            0: ''
+            - ''
+            - section_margin_edge_to_edge
```

Comment:
I'm guessing this previously manually modified and `drush config:export` reformatted it.